components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3860,6 +3878,14 @@ def get_match(name):
         else:
             arrayObject = None
 
+        typedefMemberTypes = filter(lambda t: isinstance(t, IDLTypedefType), memberTypes)
+        if len(typedefMemberTypes) > 0:
```

Comment:
For example. `typedefMemberTypes` was always null, even if I change `isinstance(t, IDLTypedefType)` to `t.isTypedef()`, causing the `if len(typedefMemberTypes) > 0:` body to never be run.